Assets/Scripts/Interfaces/IMovable.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Interfaces
+{
+    public interface IMovable
+    {
+        void Move(Vector3 position);
+
+        bool FaceTarget(GameObject target, bool shouldTurn, out float difference);
+
+        bool FaceTarget(GameObject target, bool shouldTurn);
```

Comment:
Feels kinda redundant with the other FaceTarget method defined right above it, since we can just discard the out value if we don't need it

Reply:
good point